keyboards/windstudio/wind_x/r1/readme.md

```diff
@@ -0,0 +1,26 @@
+# WindX R1 by Windstudio
+
+![Windstudio](https://windstudio.store)
```

Comment:
This doesn't appear to be an image.

Reply:
Question here: should it be the logo of the studio or an image of the keyboard?

Reply:
In the meantime I have linked the studio logo, obviously I can change as required.

Reply:
Checked other keyboards, I've linked a picture of the keyboard.